komal/Learnt-useState/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react'
+
+const App = () => {
+  const state = useState()
+
+  const [Count, setCount] = useState(0)
```

Comment:
```suggestion
  const [count, setCount] = useState(0)
```
Variable names should start with lowercase.